client/hotrod-client/pom.xml

```diff
@@ -295,6 +295,10 @@
                      </properties>
                      <parallel>none</parallel>
                      <groups>tracing</groups>
+                     <systemPropertyVariables>
+                        <!-- differentiate the log file -->
+                        <infinispan.module-suffix>${infinispan.module-suffix}-serial</infinispan.module-suffix>
```

Comment:
Should we call it -tracing instead of -serial ?

Reply:
The idea is to put there all the tests that are more critical / flaky in the long term. I don't know what you think about it.

Reply:
`serial` at the moment is the id of the maven plugin execution, in case we should change both I think. Let me know what you prefer @wburns, for me it's the same.

Reply:
I guess it is fine.